openapi/components/schemas/SubscriptionOrOneTimeSale.yaml

```
@@ -0,0 +1,10 @@
+type: object
+description: Subscription details.
+discriminator:
+  propertyName: orderType
+  mapping:
+    subscription-order: ./Subscription.yaml
+    one-time-order: ./OneTimeSale.yaml
+anyOf:
```

Schemas referenced here are mutually exclusive, as they don't intersect in terms of a required property `orderType` definition.
```suggestion
oneOf:
```